docs/howto/custom-template-tags.txt

```diff
@@ -804,6 +804,20 @@ Notes:
   ``django.template.TemplateSyntaxError`` if they receive the wrong number or
   type of arguments.
 
+* You can implement a ``stream()`` method if your renderer needs to iterate.
```

Comment:
Maybe it should be "if you rendered can iterate" instead of "needs to"? Some explanation of the advantages of implementing the `stream()` method would be helpful.